This patch file switches Projucer to GPL mode (without ROLI account requirement) and uses FST instead of the VST2 SDK.

--- ./Vendor/juce/modules/juce_audio_plugin_client/VST/juce_VST_Wrapper.cpp	(date 1556720717000)
+++ ./Vendor/juce/modules/juce_audio_plugin_client/VST/juce_VST_Wrapper.cpp	(date 1557905743224)
@@ -92,8 +92,8 @@
 // field in the Projucer. The VST2 SDK can be obtained from the
 // vstsdk3610_11_06_2018_build_37 (or older) VST3 SDK or JUCE version 5.3.2. You
 // also need a VST2 license from Steinberg to distribute VST2 plug-ins.
-#include "pluginterfaces/vst2.x/aeffect.h"
-#include "pluginterfaces/vst2.x/aeffectx.h"
+#include "fst/aeffect.h"
+#include "fst/aeffectx.h"
 }
 
 using namespace juce;

--- ./Vendor/juce/extras/Projucer/JuceLibraryCode/AppConfig.h	(date 1556720717000)
+++ ./Vendor/juce/extras/Projucer/JuceLibraryCode/AppConfig.h	(date 1557905152055)
@@ -32,7 +32,7 @@
 // BEGIN SECTION A
 
 #ifndef JUCER_ENABLE_GPL_MODE
- #define JUCER_ENABLE_GPL_MODE 0
+ #define JUCER_ENABLE_GPL_MODE 1
 #endif
 
 // END SECTION A
